core/templates/components/forms/schema-based-editors/schema-based-list-editor.directive.ts

```diff
@@ -81,6 +84,34 @@ angular.module('oppia').directive('schemaBasedListEditor', [
           return false;
         };
 
+        var initializeoldToNewAnswerChoicesMapping = function() {
+          if ($scope.oldToNewAnswerChoicesMapping !== undefined) {
+            for (var i = 0; i < $scope.localValue.length; i++) {
+              $scope.oldToNewAnswerChoicesMapping.
+                newToOldAnswerChoicesPostion.push(i);
+            }
+          }
+        };
+        var updateoldToNewAnswerChoicesMappingOnDeleteElement =
```

Comment:
Let's change to updateAnswerChoicesMappingOnDeleteItem -- a bit shorter, and allows you to get the `function(index)` on this line.

Reply:
done